kotlin/pom.kts

```diff
@@ -1,29 +1,21 @@
 project {
     name = "Polyglot :: Kotlin"
-    parent = "io.takari.polyglot:polyglot:0.3.3-SNAPSHOT"
+    groupId = "io.takari.polyglot"
     artifactId = "regular-project"
+    version = "0.3.3-SNAPSHOT"
     packaging = jar
 
     val junitVersion = 4.12
-
-    properties {
-        "project.build.sourceEncoding" assign "UTF-8"
-        "junit.version" += 4.12
-        "kotlin.version" assign "1.1.61"
-    }
+    val kotlinVersion = "1.1.61"
 
     dependencies {
-        compile("org.jetbrains.kotlin:kotlin-stdlib:" + it["kotlin.version"])
-        compile(it.groupId + ":polyglot-common:" + it.version)
-                .exclusions("org.slf4j:jul-logger")
+        compile("org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion")
+        compile("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.0.0")
```

Comment:
this is just to provide an example of exclusions in pipeline style, no real need in this dependency